modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -186,11 +197,11 @@ public static real_t Cosh(real_t s)
         /// <returns>The resulting value of the interpolation.</returns>
         public static real_t CubicInterpolate(real_t from, real_t to, real_t pre, real_t post, real_t weight)
         {
-            return 0.5f *
-                    ((from * 2.0f) +
-                            (-pre + to) * weight +
-                            (2.0f * pre - 5.0f * from + 4.0f * to - post) * (weight * weight) +
-                            (-pre + 3.0f * from - 3.0f * to + post) * (weight * weight * weight));
+            real_t ww = weight * weight;
```

Comment:
Make and test this optimization in the core engine, too. We want to keep these in sync.

Reply:
Ok tested and it's also a micro optimization, I will make the change.